.sail/Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM codercom/ubuntu-dev-go:latest
 
+RUN sudo apt-get update
```

Comment:
Any `apt-get update` should be combined with an `apt-get install` in a single `RUN` operation.

See [dockerfile best practices - RUN](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run)